src/styles/_stars.scss

```
@@ -0,0 +1,29 @@
+.stars {
+  @include center;
+
+  line-height: 16px;
+  width: 100%;
+  margin-top: 17px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    justify-content: space-between;
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+  }
```

add background-position: center;
```suggestion
  &__star {
    width: 16px;
    height: 16px;
    margin-right: 4px;
    justify-content: space-between;
    background-image: url('../images/star-active.svg');
    background-repeat: no-repeat;
    background-position: center;
  }
```